chat-client/src/client/chat.ts

```diff
@@ -15,6 +15,7 @@ import {
     SEND_TO_PROMPT,
     SendToPromptMessage,
     UiMessage,
+    DISCLAIMER_ACKNOWLEDGED,
```

Comment:
Looks like we need to publish latest `@aws/chat-client-ui-types` to npm to expose missing types

Reply:
Yeah, I'm aware. See the second bullet point in disclaimer.